### PR TITLE
Update views, add agenda, add favorite for my artists, add javascript

### PR DESCRIPTION
**Pull request by JonasJallet**
sorry Thomas for this big PR :/

### CODE REVIEW

File: src/View/Agenda/index.html.twig [hunk 1]
```diff
@@ -1,28 +1,75 @@
...
+
+	{% for favorite in favorites %}
+	<div class="mx-4">
+		<h3>
+			{{ favorite.date|format_datetime(locale='fr', pattern="MMMM yyyy")|capitalize }}
+		</h3>
+	</div>
+		<div class="row justify-content-center">
+			<div class="col-12 col-md-9 col-lg-7 col-xl-6 col-xxl-5">
+				<div onclick="toggle()" class="card mb-5 card-cover overflow-hidden border-0 text-bg-white rounded-4 shadow-lg d-flex flex-row shadow-lg">
```
Comment (aldaitzwild) [inline src/View/Agenda/index.html.twig]: No you should use an `addEventListener()` to handle the click and not the onclick attribute

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Agenda/index.html.twig (hunk 1 above)
Comment (JonasJallet) [reply src/View/Agenda/index.html.twig]: i have delete onclick

File: public/assets/js/agenda-plus.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function toggle() {
+    let visibleArray = (document).getElementsByClassName("visible");
+    for (let i = 0; i < visibleArray.length; i++)
+        visibleArray.item(i).classList.toggle('d-none');
+}
```
Comment (aldaitzwild) [inline public/assets/js/agenda-plus.js]: So now we have a function never used ... :-(. Is there a reason to keep it ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: public/assets/js/agenda-plus.js (hunk 2 above)
Comment (JonasJallet) [reply public/assets/js/agenda-plus.js]: ok.. i have delete all elements about js script for agenda

File: src/View/Artist/search.html.twig [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
-				</div>
-			{% endfor %}
-		</div>
+	<div id="allArtists" class="container marketing">
+		<h3>Suggestions</h3>
+		<div class="row">
+			{% for artist in artists %}
+				<div class="col-6 col-lg-4 col-xl-3 text-align-center">
+					<img src="{{ artist.image_artist }}" class="bd-placeholder-img rounded-circle shadow-lg" role="img" preserveaspectratio="xMidYMid slice" focusable="false">
+				</img>
```
Comment (aldaitzwild) [inline src/View/Artist/search.html.twig]: `` tags doesn't close : https://www.w3schools.com/tags/tag_img.asp

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Artist/search.html.twig (hunk 3 above)
Comment (JonasJallet) [reply src/View/Artist/search.html.twig]: delete search page not used

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**